content/en/plugins/plugin-k8s-custom-resource-status.md

```diff
@@ -326,7 +359,8 @@ spinnaker:
                   type: Reconciling
 ```
 
-These properties apply to all custom resource kinds you deploy, `Foo` and `Bar` in this case. If you deploy different kinds with different statuses, you should declare per kind like in `Example 1.1`. In this case, the plugin marks the deployment as unavailable since that matches your custom resource.
+These properties apply to all custom resource kinds you deploy through Spinnaker. If you deploy different kinds with different statuses, you should declare per kind like in `Example 2.1`. In this case, the plugin marks the deployment as ready since that
```

Comment:
These are default conditions for *ALL* CRDs unless explicitly overridden as in the examples below.